src/OWASPZAPDotNetAPI/NugetPackaging/OWASPZAPDotNetAPI.nuspec

```diff
@@ -2,17 +2,20 @@
 <package >
   <metadata>
     <id>OWASPZAPDotNetAPI</id>
-    <version>1.0.0</version>
+    <version>2.6.0</version>

```

Comment:
Do you really want to make this correspond to the ZAP version? What if a bug is found in the api implementation itself? Then it'd be out of sync again.